### DISCUSSION TITLE
Italian (Italiano)

### DISCUSSION THREAD

**Discussion by smaciullator** [sovenor/bitcoin-rocks#38] [Translations]
Hi everyone, i'll slowly translate everything on the repo, but feel free to join and help me

**Comment by sovenor** [comment]
Amazing, thank you so much for getting involved! I'll be reviewing your initial translation PR shortly.

**Comment by sovenor** [comment]
Italian is now available across the entire website. You can select it in the top language bar on any page. If your default browser language is set to Italian, it will automatically choose that for you. 

Thank you so much for your contributions @smaciullator!
